<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subsectionParamDialog</class>
 <widget class="QDialog" name="subsectionParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>分量参数</string>
  </property>
  <widget class="QDoubleSpinBox" name="Iregion0SubsectionDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Iregion0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>54</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Iregion1</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Iregion1SubsectionDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
